azure-pipeline.yml

```diff
@@ -0,0 +1,48 @@
+trigger:
+  branches:
+    include:
+    - master
+  tags:
+    include:
+    - '*'
+
+strategy:
+  matrix:
+    mac:
+      imageName: 'macos-10.13'
+    windows:
+      imageName: 'vs2017-win2016'
+
+pool:
+  vmImage: $(imageName)
+
+steps:
+
+- task: NodeTool@0
+  displayName: 'Use Node 10.x'
+  inputs:
+    versionSpec: 10.x
+
+- task: Npm@1
+  displayName: 'npm install'

```

Comment:
can use https://docs.microsoft.com/en-us/azure/devops/pipelines/caching/?view=azure-devops to improve npm install perf on hosted agent

Reply:
Good point, that is exactly what I want.

---
In reply to: [361399894](https://github.com/docascode/vscode-docs-build/pull/6#discussion_r361399894) [](ancestors = 361399894)